ethexe/contracts/src/Router.sol

```diff
@@ -272,8 +272,9 @@ contract Router is IRouter, OwnableUpgradeable, ReentrancyGuardTransient {
 
     function commitBatch(
         Gear.BatchCommitment calldata _batchCommitment,
-        Gear.SignatureType _signatureType,
-        bytes[] calldata _signatures
+        uint256 _signatureRX,
+        uint256 _signatureRY,
+        uint256 _signatureZ
```

Comment:
Could we concat this in one structure?

Reply:
make sense

Reply:
wdym but incorrect validatos? you mean malicious validators?

Reply:
https://github.com/gear-tech/gear/blob/80e008b2d0240a5eda0e440305672bc15ca7b373/ethexe/sequencer/src/lib.rs#L280-L295

Reply:
We don't listen to unauthenticated messages